docs/source/user_guide/01a_filestructure.rst

```diff
@@ -0,0 +1,63 @@
+
+Recommended file structure
```

Comment:
How about maybe... "How to structure your class files" or something like that? "Recommended" makes it sound optional, but it's really not -- it can be customized a bit, but you still do have to have a very specific file structure.